src/components/common/ModalPicker.js

```diff
@@ -22,19 +22,17 @@ class ModalPicker extends Component {
             <Picker
               style={styles.picker}
               selectedValue={this.state.selectedItem}
-              onValueChange={(value, index) => this.onValueChange(value, index)}
-              mode="dropdown"
+              onValueChange={this.onValueChange}
+              mode="dialog"
               itemStyle={styles.pickerItem}
             >
-              <Picker.Item label="Does not repeat" value="1" />
-              <Picker.Item label="Every Day" value="2" />
+              {this.state.items.map(item => {
+                return <Picker.Item label={item.name} value={item.value} />;
```

Comment:
`key`prop missing: https://reactjs.org/docs/lists-and-keys.html

Reply:
fixed